openlattice.yaml

```diff
@@ -1559,6 +1581,61 @@ paths:
                 type: object
                 additionalProperties:
                   type: object
+    put:
+      summary: Creates a new set of associations.
+      operationId: createAssociationSet
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: "#/components/schemas/dataEdgeKey"
+      tags:
+        - data
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      responses:
+        "200":
+          description: "An Entity Set definition"
+          content:
+            application/json:
+              schema:
+                type: integer
+    patch:
+      summary: Replace Association Data
```

Comment:
This is so nitpicky that I almost don't even want to say it and yet I am going to: Some summaries use verb forms like "Replaces" or "Deletes" and some use forms like "Replace" and "Create", and if we are going to make this an openly available description of the apis, I think it is more professional to be consistent. But like, it is the smallest of concerns.